modules/apps/headless/content-space/content-space-headless-apio-api/bnd.bnd

```diff
@@ -0,0 +1,7 @@
+Bundle-Name: Liferay Content Space Headless Apio API
```

Comment:
remove apio, from folder and bundle names. Should be just content-space (or what brian asked headless-content-space)